server/index.js

```diff
@@ -2,15 +2,38 @@ require('dotenv/config');
 const express = require('express');
 const staticMiddleware = require('./static-middleware');
 const errorMiddleware = require('./error-middleware');
+const axios = require('axios');
```

Comment:
Cool. I guess you need this so you can make requests to the external API from Node. Node has an http module for making requests, but it's a pain to use.